src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int OPERATION_POSITION = 0;
+    private static final int QUANTITY_POSITION = 1;
+    private static final String COMMA = ",";
+    private static final String SEPARATOR = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder inputData = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                inputData.append(value).append(SEPARATOR);
+                value = reader.readLine();
```

Comment:
Consider handling empty lines in the input file. If an empty line is encountered, `operationName` will have a length less than expected, leading to an `ArrayIndexOutOfBoundsException` when accessing `operationName[QUANTITY_POSITION]`.